src/get-glamor-classname.js

```diff
@@ -25,29 +25,21 @@ function extractGlamorStyles(className = '') {
 
 export default getGlamorClassName
 
-function getGlamorClassName(styles, props, cssOverrides, theme, context) {
+function getGlamorClassName({
+  styles,
+  props,
+  cssOverrides,
+  theme,
+  context,
+}) {
   const {mappedArgs, nonGlamorClassNames} = handleStyles(
-    styles,
+    [...styles, ...(props.className || '').split(' '), cssOverrides],
     props,
     theme,
     context,
   )
-  const {
```

Comment:
I realized that most of this stuff was just calling into `handleStyles` and I could just call that once with the joined style args and that should give us the same effect. By "should," I mean, it would if the tests were passing 🙃